frontend/src/components/CreationModal/CreationModal.js

```diff
@@ -0,0 +1,124 @@
+    const handleSelectCategory = (g) => {
+        setCategory(g);
+    }
+
+  return (
+    <Modal
+      dialogClassName={styles.width}     
+      contentClassName={styles.height} 
+      show={show}
+      onHide={onClose}
+      animation={false}
+      centered
+      >
+          <Modal.Body>
+          <div>
+              <CloseButton className = {styles.close_button} onClick={onClose}/>   
+          </div>
+          <div className={styles.container}>
+            <div className={styles.title}>
+                <h1>Create a Group</h1>
+            </div>
+            <div className={styles.body}>
+                <form>
+                    <div className={styles.single_box}>
+                        <label>Group Name</label>
+                        <input type="text" className="form-control"></input>
+                    </div>
+                    <div className={styles.double_box}>
+                        <div className={styles.flex}>
+                            <label>Date</label>
+                            <input type="text" className="form-control"></input>
+                        </div>
+                        <div className={styles.flex}>
+                            <label>Time</label>
+                            <input type="text" className="form-control"></input>
+                        </div>
+                    </div>
+                    <div className={styles.single_box}>
+                        <label>Location</label>
+                        <input type="text" className="form-control"></input>
+                    </div>
+                    <div className={styles.double_box}>
+                        <div className={styles.flex}>
+                            <label>Category</label>
+                            <div className="form-control">
+                                <DropdownButton
+                                    className={styles.dropdown_container_button}
+                                    menuVariant="light"
+                                    title={category}
+                                    id="dropdown-menu-align-right"
+                                    size="sm"
+                                    flip="true"
+                                    bsPrefix={styles.dropdown_button2}
+                                    onSelect={handleSelectCategory}
+                                >
+                                    <Dropdown.Item eventKey="female">Food</Dropdown.Item>
+                                    <Dropdown.Item eventKey="male">Hike</Dropdown.Item>
+                                    <Dropdown.Item eventKey="other">Games</Dropdown.Item>
+                                    <Dropdown.Item eventKey="female">Drinks</Dropdown.Item>
+                                    <Dropdown.Item eventKey="male">Virtual</Dropdown.Item>
+                                    <Dropdown.Item eventKey="other">Sports</Dropdown.Item>
```

Comment:
What are all these eventKey=male, female, other? Idk what it is so I'm not gonna ask you to change it, I trust it is functioning fine

Reply:
i think they're categories he came up with

you're going to have to change it when you implement categories but its fine for now

Reply:
yeah sorry i meant specifically why does Hike have eventKey="male" and Food has eventKey="female"

Reply:
ah my bad i didnt change the event keys only the text. fixed